### PR TITLE
Switch automating LTS version to renovate

### PR DESCRIPTION
**Pull request by timja**
the powershell update cli is messing with the casing of the encoding which is failing spotless.

this is much simpler anyway

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -9,5 +9,25 @@
   "labels": [
     "dependencies"
   ],
+  "packageRules": [
+    {
+      "packagePatterns": ["org.jenkins-ci.main:jenkins-war"],
+      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]+$/",
+      "registryUrls": ["https://repo.jenkins-ci.org/public/"]
```
Comment (timja) [inline .github/renovate.json:16]: registry url extraction seems to be broken for regex custom managers

File: .github/renovate.json [hunk 2]
```diff
@@ -9,5 +9,25 @@
   "labels": [
     "dependencies"
   ],
+  "packageRules": [
+    {
+      "packagePatterns": ["org.jenkins-ci.main:jenkins-war"],
+      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]+$/",
```
Comment (timja) [inline .github/renovate.json:15]: only lts versions

**Comment by rantoniuk** [comment]
@timja are there any plans for this to be part of https://github.com/jenkinsci/.github/blob/master/.github/renovate.json ?

**Comment by timja** [comment]
> @timja are there any plans for this to be part of [jenkinsci/.github@`master`/.github/renovate.json](https://github.com/jenkinsci/.github/blob/master/.github/renovate.json?rgh-link-date=2025-06-21T20%3A07%3A05Z) ?

Contributions are welcome 👍, although thats not a template one as far as I know I think its for that repo.

Normally you put an onboarding default in a file called `renovate-config.json`.

**Pull request merged**